src/seedu/addressbook/AddressBook.java

```diff
@@ -203,13 +203,17 @@ public static void main(String[] args) {
         processProgramArgs(args);
         loadDataFromStorage();
         while (true) {
-            String userCommand = getUserInput();
-            echoUserCommand(userCommand);
-            String feedback = executeCommand(userCommand);
-            showResultToUser(feedback);
+            doInput();
         }
     }
 
+	private static void doInput() {
```

Comment:
You should add a header comment as it is not obvious to a reader what the function is doing. Also, please set your IDE to convert all tabs to 4 spaces.